WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -275,6 +283,7 @@ def nearby_competitions
     @competition = Competition.new(competition_params)
     @competition.valid? # We only unpack dates _just before_ validation, so we need to call validation here
     @competition_admin_view = params.key?(:competition_admin_view) && current_user.can_admin_results?
+    @nearby_competitions = get_nearby_competitions(@competition)
```

Comment:
Since this was done inside the view, and we call this action for ajaxing (I think), I needed to put the call here too.